ansible/convert-to-docksucker.yml

```diff
@@ -35,11 +30,17 @@
       repo: 'https://github.com/calr0x/OT-DockSucker.git'
       dest: /root/OT-DockSucker
       force: yes
-  
-  - name: Copy node config file to server
+
+  - name: Clone OT-Settings git repository
+    git:
+      repo: 'https://github.com/calr0x/OT-Settings.git'
+      dest: /root/OT-Settings
+      force: yes
+
+  - name: Copy config file
     copy:
-      src: ~/OT-DockSucker/data/config.sh
```

Comment:
We want to keep ~ as the src as this allows non-root control server users. Let me know if this doesn't make sense!

Reply:
makes total sense !